gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/entities/StageableTableMetadata.java

```diff
@@ -92,6 +98,12 @@
    */
   public static final String SOURCE_DATA_PATH_IDENTIFIER_KEY = "source.dataPathIdentifier";
 
+  /**
+   * Attributes like "avro.schema.literal" are usually used in offline system as the source-of-truth of schema.
+   * This configuration's value should the key name that users expects to preserve to schema string if necessary.
```

Comment:
"should the key name" -> "should be the key name" ?

Reply:
Fixed.